src/main.rs

```diff
@@ -63,5 +67,11 @@ fn run_new_user(args: Args) -> Result<(), Error> {
 	let public_key = args.arg_public_key.unwrap();
 	let public_key_path = Path::new(&public_key);
 	let name = args.arg_name.unwrap();
-	proton_cli::new_user(&public_key_path, name)
+	proton_cli::new_user(&public_key_path, &name)
+}
+
+fn run_id_user(args: Args) -> Result<(), Error> {
+	let private_key = args.arg_private_key.unwrap();
+	let _ = try!(proton_cli::id_user(&private_key));
```

Comment:
I think you can change this to a `map(|_| ())`. It just looks weird having an Ok(()) as the return value, even if you can easily tell that there's a try! doing error handling before it.